lib/user.rb

```diff
@@ -0,0 +1,34 @@
+require "bcrypt"
+
+class User
+  attr_reader :id, :email
+  def initialize(id:, email:)
+    @id = id
+    @email = email
+  end
+
+  def self.create(email:, password:)
+    encrypted_password = BCrypt::Password.create(password)
+    if ENV["ENVIRONMENT"] == "test"
```

Comment:
Same as above! And probably now the best idea is to extract that function to a helper that you can use in both classes you decided to implement!